Loadable.svelte

```diff
@@ -125,7 +126,17 @@
     state = STATES.SUCCESS
     component = LOADED.get(loader)
   } else {
-    onMount(load)
+    onMount(() => {
+      load()
+      if (unload && unload !== 'false') {
```

Comment:
```suggestion
      if (unload) {
```

Reply:
The thing is `unload` (or `unloadOnDestroy`) could either be `'true'`, `true`, `'false'`, `false` or a `function` and in the case of it being text like `unloadOnDestroy="false"` this would be truthy!

I couldn't find a better way to coerce text true/false to a boolean :smile:

Reply:
Oh, I see! I was thinking about only accepting actual boolean values, so we could use the boolean shorthand: `<Loadable ... unloadOnDestroy />`

Reply:
Or, if we go through the callback path, `<Loadable ... unloadOnDestroy={() => ...} />`